spec/features/heroku_spec.rb

```diff
@@ -22,7 +22,7 @@
     buildpacks_path = "#{project_path}/.buildpacks"
     buildpacks = IO.read(buildpacks_path)
 
-    expect(buildpacks).to include("https://github.com/platanus/heroku-buildpack-bower.git")
+    expect(buildpacks).to include("https://github.com/heroku/heroku-buildpack-nodejs")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Line is too long. [86/80]